src/mediarithmics/plugins/audience-feed-connector/AudienceFeedConnectorBasePlugin.ts

```diff
@@ -57,6 +63,31 @@ abstract class GenericAudienceFeedConnectorBasePlugin<
     return response.data;
   }
 
+  async fetchUserAgentIdentifierRealms(datamartId: string): Promise<Array<UserAgentIdentifierRealmSelectionResource>> {
```

Comment:
UserAgentIdentifierRealmSelectionResourcesResponse is already a list, is that a mistake ?

Reply:
It is a DataListResponse, not a list

Reply:
I red UserAgentIdentifierRealmSelectionResourcesResponse instead of UserAgentIdentifierRealmSelectionResource, my bad